javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/EndpointFeatures.qll

```diff
@@ -145,6 +163,67 @@ module FunctionBodies {
   }
 }
 
+/** This module provides functionality for getting the neighborhood scope body feature associated with a neighborhood around an AST node. */
+module NeighborhoodBodies {
```

Comment:
Can the constants 128 and 256 be pulled out and passed as parameters? We may want to add several features with different values, to cover narrower and broader scopes.